avalanche-ops/src/aws/spec.rs

```diff
@@ -63,7 +63,10 @@ pub struct Spec {
     pub create_dev_machine: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dev_machine: Option<DevMachine>,
-
+    /// Filepath to an optional directory or file containing bash scripts.
+    /// These scripts will be invoked on the dev machine upon startup.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub dev_machine_scripts_path: Option<Box<std::path::Path>>,
```

Comment:
Let's do `String` for now, to be consistent with other fields?

Reply:
I think we should use a [PathBuf](https://doc.rust-lang.org/stable/std/path/struct.PathBuf.html), which is similar to a `String`, per advice from @rkuris. If the path is invalid it will fail at deserialization time instead of later, which is preferable.